src/views/CollectionIndex/scss/index.scss

```diff
@@ -38,7 +38,7 @@
   background: $light-grayish-orange;
   padding: 16px 35px;
   font-size: 14px;
-  border-radius: 7px;
+  border-radius: 25px;
   box-shadow: 0px 3px 6px #00000029;
```

Comment:
You can use a variable for this whole expression `0px 3px 6px #00000029`

Remember to change the `box-shadow` below too.

Reply:
Done